app/views/documents/_header.html.erb

```diff
@@ -24,3 +24,8 @@
                                  topics: topics,
                                  primary_mainstream_category: primary_mainstream_category,
                                  specialist_tag_finder: specialist_tag_finder %>
+
+<% if document.historic? %>
+<%= render('documents/history_notice', type: document.format_name,
+                                       government_name: document.government.try(:name)) %>
```

Comment:
Assuming that `#historic?` will always rely on the presence of a government seems like bad practice, as does having the partial know about the internal implementation of that method.